content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,191 @@
+
+![Jabber.org](https://www.jabber.org/jabber.png)
+
+jabber.org migrates its server to Prosody. [Stay informed!](https://www.jabber.org/notices.html)
+
+yax.im rolled out [new anti-spam measures](https://yaxim.org/blog/2020/05/12/new-anti-spam-measures/). You may be interested in the '[The Jabber Spam Fighting Manifesto](https://github.com/JabberSPAM/jabber-spam-fighting-manifesto)', too.
+
+## Libraries
+
+The XMPP library for Python 3.7+, [Slixmpp](https://lab.louiz.org/poezio/slixmpp), was [updated to version 1.5.0](https://mail.jabber.org/pipermail/jdev/2020-May/090424.html), with minor releases 1.5.1 and 1.5.2. The [changelog]( https://lab.louiz.org/poezio/slixmpp/-/tags/slix-1.5.0) shows improvements of numerous XEPs and preliminary support for OMEMO.
+
+Quite some time has past since the last official Smack release. Now there are some very early alpha releases of Smack 4.4.
+Among additional support for more than a dozen new XEPs, the new release will contain a new architecture for modular transports, as well as a modified API that applies the builder pattern to stanza types.
+A date for the final release is not yet known, but it is now time to give feedback to make sure to get your changes into the next stable Smack release!
+[Read the detailed list of changes].(https://github.com/igniterealtime/Smack/wiki/Smack-4.4-Readme-and-Upgrade-Guide).
+
+## Google Summer of Code
+
+Since beginning of May students all over the world are working on many open-source projects within the Google Summer of Code Season. We would like to introduce the students who work on [GSoC XMPP projects](https://summerofcode.withgoogle.com/organizations/5474262808264704/#6018598289539072) and share their first blog posts:
+
+[Aditya Borikar](https://adiaholic.github.io/gsoc2020/2020/05/16/Chapter-0-Introduction.html) is working on WebSocket support for Smack. Blog history: 
+- [Chapter 1: Handshake](https://adiaholic.github.io/gsoc2020/2020/05/24/Chapter-1-Handshake.html)
+- [Chapter 2: Modular Shift](https://adiaholic.github.io/gsoc2020/2020/05/31/Chapter-2-Modular-Shift.html)
+
+[Anmol (wolfie_anmol)](https://wolfieanmol.github.io/gsoc-blog/) is working on implementing Real Time Texting in Dino ([XEP-0301](https://xmpp.org/extensions/xep-0301.html)). Blog history:
+- [GSoC 2020 begins - Introduction to RTT](https://wolfieanmol.github.io/gsoc-blog/introduction/)
+- [RTT stanza and message comparision](https://wolfieanmol.github.io/gsoc-blog/rtt-stanza-and-message-comparision/)
+
+Thank you for joining GSoC XMPP projects and keep up the good work! To be continued.
+
+## Extensions and specifications
+
+### Updated
+
+- Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations
```

Comment:
All this Markdown code does not do what's intended, I think.

```
- Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations for XMPP)
* Make Blake2b-256 SHOULD and Blake2b 512 MUST (rion)

```
- Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations for XMPP)
* Make Blake2b-256 SHOULD and Blake2b 512 MUST (rion)

Should be:
```
- Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations for XMPP)
  - Make Blake2b-256 SHOULD and Blake2b 512 MUST (rion)

```
- Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations for XMPP)
  - Make Blake2b-256 SHOULD and Blake2b 512 MUST (rion)
Or:
```
* Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations for XMPP)
  * Make Blake2b-256 SHOULD and Blake2b 512 MUST (rion)
```
* Version 0.4.0 of [XEP-0414](https://xmpp.org/extensions/xep-0414.html) (Cryptographic Hash Function Recommendations for XMPP)
  * Make Blake2b-256 SHOULD and Blake2b 512 MUST (rion)